src/components/SchemaView/SchemaViewMain.vue

```diff
@@ -655,10 +657,39 @@ export default {
       this.reloadSchema();
     },
 
+
+    enterRenameNodeTableMode(newTableName) {
```

Comment:
If the code here is not being used, you should remove the code instead of commenting out